content/applications/finance/fiscal_localizations/austria.rst

```diff
@@ -0,0 +1,137 @@
+     - Adds the SAF-T export.
+     - Enterprise
+
+.. image:: austria/austria-modules.png
+   :alt: Modules for the Austrian localization
+
+Financial Reports
+=================
+
+.. seealso::
+   :doc:`../accounting/reporting`
+
+Currently the following localized reports are available:
+
+  - Balance sheet according to § 224 UGB
+  - Profit and loss according to § 231 UGB (Gesamtkostenverfahren)
+
+To populate the individual lines of these reports the mapping from
+your company's chart of accounts to the SAF-T accounts is used.
+See :ref:`the information about the chart of accounts below <austria/SAFT/chart_of_accounts>`
+
+Standard Audit File for Tax (SAF-T)
+===================================
+
+The Austrian tax office may request a SAF-T report on a on-demand basis.
+The :guilabel:`Austrian SAF-T Export` module provides the ability to export a SAF-T document
+(in XML format).
+
+Configuration
+-------------
+
+The following explains how to ensure that all the information that is needed
+for the SAF-T export is available. In case missing information is detected during the export process
+a warning message will appear that lists the missing information.
+
+Company
+~~~~~~~
+
+Under :guilabel:`Settings --> General Settings`, in the **Companies** section, click
+:guilabel:`Update Info` and fill in the following information:
+
+- Fill in the :guilabel:`Address` and :guilabel:`Phone` fields.
+  For the address please enter (at least) :guilabel:`Street`, :guilabel:`City`, :guilabel:`ZIP`
+  and :guilabel:`Country`.
+- Provide your company's tax ID in the :guilabel:`Company ID` field.
+- If you have one, provide your :abbr:`UID-Nummer (Umsatzsteueridentifikationsnummer)`
+  in the :guilabel:`Tax ID` field number, including the country prefix.
+
+You must have at least one **contact person** linked to your company in the **Contacts** app:
+
+- If no **contact person** is linked, create a new one by clicking :guilabel:`New`,
+  set it as :guilabel:`Individual`, and select your company in the :guilabel:`Company name` field.
+- At least one of the :guilabel:`Phone` and :guilabel:`Mobile` fields has to be filled in
+  for the contact person.
+
+Under :guilabel:`Settings --> Accounting`, in the **Austrian localization** section,
+set the following information:
+
+- :guilabel:`ÖNACE-Code`
+- :guilabel:`Profit Assessment Method`
+
+.. _austria/SAFT/chart_of_accounts:
+
+Chart of accounts
+~~~~~~~~~~~~~~~~~
+
+.. seealso::
+   :doc:`../accounting/get_started/chart_of_accounts`
+
+The Austrian SAF-T specification defines a chart of accounts (COA). All accounts that are relevant
+for the SAF-T export have to be annotated with a fitting account from this COA.
+
+The needed mapping information has to be supplied via tags on the accounts.
+Applying the tag `1000` on an account (virtually) maps it to the account with code `1000`
+in the SAF-T COA.
+Similarly any number can be used instead of `1000` to map an account to any account in the SAF-T COA
```

Comment:
Similarly, any number can be used instead of `1000` to map an account to any account in the SAF-T COA